addons/web/static/tests/views/fields/many2one_avatar_field_tests.js

```diff
@@ -87,7 +87,7 @@ QUnit.module("Fields", (hooks) => {
             target,
             '.o_m2o_avatar > img[data-src="/web/image/user/17/avatar_128"]'
         );
-        assert.containsOnce(target, '.o_field_many2one_avatar > div[data-tooltip="Aline"]');
+        assert.containsOnce(target, ".o_field_many2one_avatar > div");
```

Comment:
why?

Reply:
We cannot guarantee the existence of the tooltip since the xpath removes it when applied